signature-aggregator/api/api.go

```diff
@@ -147,6 +147,7 @@ func signatureAggregationAPIHandler(
 			msg := "Failed to aggregate signatures"
 			logger.Warn(msg, zap.Error(err))
 			writeJSONError(logger, w, msg)
+			return
```

Comment:
I know this wasn't you, but should we also return after writing the JSON error on line 137?

Reply:
good catch. the one I added already was because I was hitting it in my testing, but I didn't think to look for more. thanks! addressed in https://github.com/ava-labs/awm-relayer/pull/415/commits/acb3e80a838bbf3c3305d93179e55dcad4df7a46